build-package.sh

```diff
@@ -598,7 +633,32 @@ if [ "${TERMUX_INSTALL_DEPS-false}" = "true" ] || [ "${TERMUX_PACKAGE_LIBRARY-bi
 	}
 fi
 
-for ((i=0; i<${#PACKAGE_LIST[@]}; i++)); do
+# Remove duplicate and empty packages passed
+declare -A PACKAGES_FOUND_LIST=()
+declare -a PACKAGES_LIST=()
+
+for i in "${PACKAGES_ARGS_LIST[@]}"; do
+	if [[ -z "${i:-}" ]]; then
```

Comment:
AFAIK it is not needed to do `:-` inside `[[`, the test will not cause bailing because of `set -u`. Same applies to all places you used this.

Also AFAIK this causes some speed hit.